### PR TITLE
Update eslint-config-fusion@0.2.0 and apply import fixes

### PR DESCRIPTION
**Pull request by KevinGrandon**
This linter upgrade will include the import linter which will avoid issues like #6. Also including some work to get around the linter, such as preferring default exports,

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
 // @flow
-export {withFontLoading} from './with-font-loading';
-export {FontPlugin} from './plugin';
+import withFontLoading from './with-font-loading';
```
Comment (angus-c) [inline src/index.js]: linter prefers this pedantic style?

Comment (KevinGrandon) [reply src/index.js]: Boils down to this basically: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

I suppose we can change this to `export {default as withFontLoading} from ...`? I'll try that as well.

**Comment by angus-c** [comment]
@KevinGrandon not a big deal, just surprised.

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**